bin/p2-exec/main.go

```diff
@@ -114,6 +114,12 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+
+	// setpgid: zeroes for current process ID
```

Comment:
Nit: commentary below should be added to the code, so someone looking 3 years from now knows why this decision was made! Also we should describe why this is being done in the first place.